ccm_web/server/src/api/api.service.ts

```diff
@@ -10,9 +11,12 @@ export class APIService {
     }
   }
 
-  getGlobals (): Globals {
+  getGlobals (data: SessionData): Globals {
```

Comment:
Nit-pick: I think I'd use `sessionData` or `session` for the parameter name for clarity.

Reply:
Make sense to do it like that